### ISSUE TITLE
get_r copies remote folder structure into localdir

### ISSUE THREAD

**Issue by coreyhartley** [byteskeptical/sftpretty#28]
We have been using the following code (formerly `pysftp`) to navigate to the remote directory, to then copy its contents to a local directory:
```
sftp.chdir(remote_directory)
sftp.get_r(".", local_directory)
```

If our remote path was structured like `\path\to\remote\data\*` (where `*` is our data), and our local directory was `\path\local`, our directory structure now looks like `\path\local\path\to\remote\data\*` after using `get_r`.  We would expect to see the directory structure look like `\path\local\*` after using it.

**Comment by mawaliya** [comment]
Can we expect the same thing for remote side as well (put_d and put_r)?

**Comment by byteskeptical** [comment]
This will be fixed in PR #26. Currently trying to get all test working on Windows and MacOSX action images.

**Comment by byteskeptical** [comment]
> Can we expect the same thing for remote side as well (put_d and put_r)?

Please don't hijack other peoples issues with comments referencing your own. I've addressed your concern on the issue you created.
